src/ComScoreAnalytics.ts

```diff
@@ -40,6 +40,10 @@ export class ComScoreAnalytics {
         ns_.comScore.setPublisherSecret(configuration.publisherSecret);
         ns_.comScore.setAppName(configuration.applicationName);
         ns_.comScore.setAppVersion(configuration.applicationVersion);
+
+        if (configuration.userConsent !== undefined) {
+          ns_.comScore.setLabels({ cs_ucfr: configuration.userConsent === ComScoreUserConsent.Granted ? '1' : '0'});
```

Comment:
As the value of the enum is `1` or `0` already you don't need the check here. You can just write:
```suggestion
          ns_.comScore.setLabels({ cs_ucfr: configuration.userConsent });
```

Reply:
I want to make sure it gets set as the string value though. Will that still work without the operation?

Reply:
As the Enum is a string enum yes.